frontend/storage/utils/fetchGameData.ts

```diff
@@ -13,13 +13,27 @@ export const FetchGame = {
 			throw new Error(error);
 		}
 	},
+	lookupGames: async function() {
+		try {
+			return await ApiRequests.get<Array<Game>>(`games/`, {});
+		} catch (error) {
+			throw new Error(error);
```

Comment:
That's a lot of duplicated `try{}catch{}`. You may want to see if there's a way to remove it, perhaps make the call to `ApiRequestes.get()` guarantee an `Error()` is thrown, so you don't have to rethrow it?